support/@trustfractal/data-host/src/popup/services/FractalAccount.ts

```diff
@@ -0,0 +1,81 @@
+import environment from '@popup/Environment';
+import { MultiContext } from '@utils/MultiContext';
+import { Storage } from '@utils/StorageArray';
+import { ReplaySubject } from 'rxjs';
+
+interface Tokens {
+  scopes: string;
+}
+
+export class NotConnectedError extends Error {
+  constructor(message?: string) {
+    super(message);
+    this.name = 'NotConnectedError';
+  }
+}
+
+const NEXT_TOKENS_KEY = 'fractal-account-connector/will-accept-next-tokens';
+const TOKENS_KEY = 'fractal-account-connector/tokens';
+
+export class FractalAccountConnector extends MultiContext {
```

Comment:
Most of this class is for our old way of connecting the account. We can remove/change it now or later.

Reply:
By the old way, do you mean that we are currently using another way in the Fractal Wallet? If so, can you send me link to it.

Reply:
I mean the way the wallet is using now, but what we're moving away from. You can include this for now, but be sure to remove the unused code once fractal.id registers the identity.